cypress-tests/cypress/fixtures/merchant-create-body.json

```diff
@@ -18,19 +18,23 @@
       "city": "San Francisco",
       "state": "California",
       "zip": "94122",
-      "country": "US"
+      "country": "US",
+      "first_name": "john",
+      "last_name": "Doe"
     }
   },
-  "return_url": "https://example.com",
+  "return_url": "https://google.com/success",
```

Comment:
why? example.com was intentional and that is because we do not want  the redirection handler to scream at pages breaking.